build.gradle

```diff
@@ -153,12 +153,22 @@ dependencyManagement {
   }
 }
 
+configurations {
+  implementation {
+    exclude group: 'com.vaadin.external.google', module: 'android-json'
+  }
+}
+
 repositories {
   jcenter()
 
   maven {
     url "https://dl.bintray.com/hmcts/hmcts-maven"
   }
+
+  maven {
+    url "https://jitpack.io"
```

Comment:
worth adding a comment here that json schema library is primarily distributed through JitPack.